src/main/resources/db/schema/V1_0__schema.sql

```diff
@@ -9050,6 +9050,8 @@ create table PRISON_OFFENDER_MANAGER
     CREATED_BY_USER_ID         NUMBER           not null,
     LAST_UPDATED_DATETIME      DATE             not null,
     LAST_UPDATED_USER_ID       NUMBER           not null,
+    EMAIL_ADDRESS VARCHAR2(255),
```

Comment:
I wonder if it would be better to comment these out?? At least it proves everything works with out those columns

Reply:
Done